CHANGELOG.md

```diff
@@ -5,6 +5,26 @@
 
 # Changelog
 
+## [v8.34.1](https://github.com/nextcloud-libraries/nextcloud-vue/tree/v8.34.1) (2025-12-10)
+### 🚀 Enhancements
+* [stable8] feat(NcAppSettingsDialog): add version on the bottom and `noVersion` prop to disable it  by @ShGKme in https://github.com/nextcloud-libraries/nextcloud-vue/pull/7850
+* [stable8] fix(NcActions): introduce 'size' prop by @backportbot[bot] in https://github.com/nextcloud-libraries/nextcloud-vue/pull/7852
+* [stable8] feat(NcRichText): add 'Copy to clipboard' button for code blocks by @backportbot[bot] in https://github.com/nextcloud-libraries/nextcloud-vue/pull/7877
```

Comment:
Should be a minor release, two components from the list have changed default appearance, and might  need adjustment by app devs (so it's not a blind saturday bump)

Reply:
Looks like we missed `v8.35.0` when released `v9.3.0`